### DISCUSSION TITLE
Why must CONFIG_MP_NUM_CPUS == CONFIG_MP_MAX_NUM_CPUS  be meet when building project?

### DISCUSSION THREAD

**Discussion by cocoeoli** [zephyrproject-rtos/zephyr#63306] [Q&A] [answered]
I want to support  SMP for an AArch64 development board, but during the Twister test, I was unable to pass some test, and the error results are as follows:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/40254934/dada41fe-3d1a-4545-a51c-4fa94a19d49f)

But in fact, after I modified the CONFIG_MP_MAX_NUM_CPUS in the prj.conf file in the app directory. The use case can run normally. 
May I ask how to modify my code to pass the Twister test?

**Comment by kartben** [comment, marked as answer]
See https://github.com/zephyrproject-rtos/zephyr/pull/60652

**Reply by cocoeoli** [reply]
Thanks @kartben , I have found the reason, It is becasue MP_NUM_CPUS can not be set to muti-core for some test case. Just set the MP_MAX_NUM_CPUS can work well.
